packages/pds/src/index.ts

```diff
@@ -186,6 +186,18 @@ export class PDS {
     }
 
     app.use(health.createRouter(ctx))
+    app.get('/', function (req, res) {
```

Comment:
could we move these to a supplementary file so as not to clog up the entry file?

besides that, looks good!

Reply:
created a "basic-routes.ts" file, and pulled `/xrpc/_health` out of the `src/api/` folder as well. I guess we might want to keep all the `/xrpc/*` stuff together under `api/`, but this felt like the right thing to do because we are already attaching that router from `index.ts` and attaching two "housekeeping" routers felt wrong.